package-dev/Tests/Editor/TestFiles/main_2019_4_expected.txt

```diff
@@ -0,0 +1,33 @@
+#include <Sentry/Sentry.h>
+#include "SentryOptions.m"
+#include <UnityFramework/UnityFramework.h>
+
+UnityFramework* UnityFrameworkLoad()
+{
+    NSString* bundlePath = nil;
+    bundlePath = [[NSBundle mainBundle] bundlePath];
+    bundlePath = [bundlePath stringByAppendingString: @"/Frameworks/UnityFramework.framework"];
+
+    NSBundle* bundle = [NSBundle bundleWithPath: bundlePath];
+    if ([bundle isLoaded] == false) [bundle load];
+
+    UnityFramework* ufw = [bundle.principalClass getInstance];
+    if (![ufw appController])
+    {
+        // unity is not initialized
+        [ufw setExecuteHeader: &_mh_execute_header];
+    }
+    return ufw;
+}
+
+int main(int argc, char* argv[])
+{
+    @autoreleasepool
+    {
+		[SentrySDK startWithOptions:getSentryOptions()];
```

Comment:
mix of tabs and spaces:

```suggestion
    {
		[SentrySDK startWithOptions:getSentryOptions()];
```

Can we replace our code to add `        ` instead of `\t\t`?